<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stegencry</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="encryption_label">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>10</y>
      <width>1141</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="decrypt_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>630</y>
      <width>561</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QToolButton" name="add_image_encryption">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>10</y>
      <width>1141</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="encrypt_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>561</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionOpen_Child"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuKey">
    <property name="title">
     <string>Key</string>
    </property>
    <addaction name="actionOpen_Key_2"/>
    <addaction name="actionSet_Key_2"/>
    <addaction name="separator"/>
    <addaction name="actionGenerate_Key"/>
    <addaction name="actionSave_Key"/>
   </widget>
   <widget class="QMenu" name="menuCustom">
    <property name="title">
     <string>Custom</string>
    </property>
    <addaction name="actionCustom_Dialog"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuKey"/>
   <addaction name="menuCustom"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Child">
   <property name="text">
    <string>Open Child (for Steganography only)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionOpen_Key">
   <property name="text">
    <string>Open Key</string>
   </property>
  </action>
  <action name="actionSet_Key">
   <property name="text">
    <string>Set Key</string>
   </property>
  </action>
  <action name="actionOpen_Key_2">
   <property name="text">
    <string>Open Key</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionSet_Key_2">
   <property name="text">
    <string>Set Key-Password</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+K</string>
   </property>
  </action>
  <action name="actionGenerate_Key">
   <property name="text">
    <string>Generate Key</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSave_Key">
   <property name="text">
    <string>Save Key</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCustomization_Dialog">
   <property name="text">
    <string>Customization Dialog</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Seeking for help? Click here.</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSteganography">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Steganography</string>
   </property>
  </action>
  <action name="actionRGB_Encryption">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>RGB Encryption</string>
   </property>
  </action>
  <action name="actionImage_Encryption">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Image Encryption</string>
   </property>
  </action>
  <action name="actionCustom_Dialog">
   <property name="text">
    <string>Customization</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
